docs/operations-guide/migrating-from-h2.md

```diff
@@ -1,6 +1,13 @@
 # Migrating from using the H2 database to MySQL or Postgres
 
-If you decide to use the default application database (H2) when you initially start using Metabase, but later decide that you'd like to switch to a more production-ready database such as MySQL or Postgres, we make the transition easy for you.
+If you decide to use the default application database (H2) when you initially start using Metabase, but later decide that you'd like to switch to a more production-ready database such as MySQL or Postgres, you're in the right place.
```

Comment:
```suggestion
If you decide to use the default application database (H2) when you initially start using Metabase, but later decide that you'd like to switch to a high-availability database such as MySQL or Postgres, you're in the right place.
```

Reply:
Postgres/MySQL are not HA unless they are configured as such.